jest.config.js

```diff
@@ -0,0 +1,30 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
```

Comment:
Please remove most of the comments as they may get outdated and docs should be used as a source of truth for configs.